sitemedia/scss/pages/_search.scss

```diff
@@ -27,3 +27,12 @@
         padding: 0 spacing.$spacing-md spacing.$spacing-md;
     }
 }
+
+// Applying color per system theme
+.turbo-progress-bar {
+    height: 5px;
+    background-color: #436043;
+    @media (prefers-color-scheme: dark) {
+        background-color: #c37f97;
```

Comment:
Were you not able to get the css variables to work either? is it because of where the progress bar is added?

we should at least have comments here; maybe scss variables defined in the same place as the css variables 

I think you're missing the theme override cases here.

Reply:
Removed for now based on your comment below, we'll address it separately. I think we might want to apply themes to the `html` element instead of `body`.